library/src/main/java/com/novoda/downloadmanager/lib/DownloadsDestination.java

```diff
@@ -0,0 +1,63 @@
+package com.novoda.downloadmanager.lib;
+
+final class DownloadsDestination {
```

Comment:
this is not OO programming :x: the constants should be declared where they are used. However if you are doing to do this route at least follow the Android recommended pattern with a "contract class" like shown in the Udacity course on ContentProviders.